wrapanapi/containers/image_registry.py

```diff
@@ -1,12 +1,12 @@
-from cached_property import cached_property
-
 from wrapanapi.containers import ContainersResourceBase
 from wrapanapi.exceptions import RequestFailedException
 from wrapanapi.containers.image import Image
 
 
 class ImageRegistry(ContainersResourceBase):
     RESOURCE_TYPE = 'imagestream'
+    KIND = 'ImageStream'
```

Comment:
I note that this is differently named, perhaps we could check for something called CLASS_NAME_OVERRIDE, just seems like more effort to create another KIND for every class which is already present in the name

Reply:
I keep this as KIND since it used for "Kind" property in the Json content.